### PR TITLE
Adds more Halloween costumes

### PR DESCRIPTION
**Pull request by FuryMcFlurry**
Here goes baby's first PR. (GitKraken sucks.)
I've sprited and added some costumes.
Pictures here.
Draculass
image
Dracula
image
Lobster
image
Dr Freeze
image
Goth Coat
image

:cl: Fury McFlurry
add: Added more halloween costumes.
add: Among them are a lobster suit, a cold villain costume and some gothic clothes
/:cl:

### CODE REVIEW

**Comment by RemieRichards** [comment]
>Draculass

More like let's do the twist again.

**Comment by YakumoChen** [comment]
:+1: for Lobsterman

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -693,6 +693,33 @@
...
+	can_adjust = 0
+
+/obj/item/clothing/under/lobster
+	name = "foam lobster suit"
+	desc = "Who beheaded the college mascot?"
+	icon_state = "lobster"
+	item_state = "lobster"
+	item_color = "lobster"
+	fitted = NO_FEMALE_UNIFORM
+	can_adjust = 0
```
Comment (Cyberboss) [inline code/modules/clothing/under/miscellaneous.dm]: FALSE

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -693,6 +693,33 @@
...
+	fitted = FEMALE_UNIFORM_TOP
+	can_adjust = 0
+
+/obj/item/clothing/under/drfreeze
+	name = "doctor freeze's jumpsuit"
+	desc = "A modified scientist jumpsuit to look extra cool."
+	icon_state = "drfreeze"
+	item_state = "drfreeze"
+	item_color = "drfreeze"
+	can_adjust = 0
```
Comment (Cyberboss) [inline code/modules/clothing/under/miscellaneous.dm]: FALSE

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -693,6 +693,33 @@
...
 
+/obj/item/clothing/under/draculass
+	name = "draculass coat"
+	desc = "A dress inspired by the ancient era known as the victorian era"
+	icon_state = "draculass"
+	item_state = "draculass"
+	item_color = "draculass"
+	body_parts_covered = CHEST|GROIN|ARMS
+	fitted = FEMALE_UNIFORM_TOP
+	can_adjust = 0
```
Comment (Cyberboss) [inline code/modules/clothing/under/miscellaneous.dm]: FALSE

**Review by Cyberboss — CHANGES_REQUESTED** [review]
These aren't mapped in anywhere?

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (FuryMcFlurry) [reply code/modules/clothing/under/miscellaneous.dm]: About all the clothing code has those. I just copied the way how everything was not to mess anything up. I have to figure out for to update this Pr.

**Comment by FuryMcFlurry** [comment]
> These aren't mapped in anywhere?

Much like Kor's  costume PR these will just add costume them to ingame, maybe near halloween these will be added to the game somewhere

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**